source/_integrations/onvif.markdown

```diff
@@ -66,6 +66,7 @@ To help with development of this integration, enable `info` level logging for `h
 | Motion alarm | Binary sensor | Motion | Generic motion alarm. |
 | Field detection | Binary sensor | Motion | Polygonal field detection determines if each object in the scene is inside or outside the polygon. |
 | Cell motion detection | Binary sensor | Motion | Cell based motion detection determined by placing a grid over the video source and determining changes. |
+| Human shape detection | Binary sensor | Motion | Detection of human shapes by in camera recognition algorithm. |
```

Comment:
```suggestion
| Human shape detection | Binary sensor | Motion | Detection of human shapes by on-camera recognition algorithm. |
```
should this be on-camera, maybe? Does it mean the calculation is done on the camera?

Reply:
thank you for explaining!